src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+      <ol>
+        <li>
+          <FormattedMessage id="tnc.participate1" defaultMessage="You may only use the Site and/or Platform if You comply with all of the following:" />
+          <ol>
+            <li><FormattedMessage id="tnc.participate1a" defaultMessage="You are at least eighteen (18) years old or of legal age as determined by the laws of the country where You live (whichever is higher); and" /></li>
+            <li><FormattedMessage id="tnc.participate1b" defaultMessage="You do not violate any law or regulation as a result of using the Site and/or Platform. In this context, You agree that if You reside or are present in any jurisdiction that prohibits using the Site and/or Platform (including without limitation any of the Restricted Territories) You shall not participate in the prohibited activity." /></li>
+          </ol>
+        </li>
+        <li><FormattedMessage id="tnc.participate2" defaultMessage="The Site and/or Platform are intended only for users who are not prohibited by the laws of any applicable jurisdiction from creating Events, making Trades and Predictions, or otherwise Participating on the Platform. The Company does not intend to enable You to contravene applicable law. You represent, warrant and agree to ensure that Your use of the Site and/or Platform will comply with all applicable laws, statutes and regulations. The offering or availability of the Site and/or Platform shall not be deemed or interpreted as an offer or invitation by Us to use the Site and/or Platform, if You reside in a place in which such use is currently forbidden by law (including without limitation the Restricted Territories), or where Bodhi, in its sole discretion, elects not to offer Site and/or Platform. You shall be solely responsible for determining whether Your use of the Site and/or Platform is legal in the place where You live and/or use the Site and/or Platform. We make no representations or warranties, express or implied, concerning the legality of the Site and/or Platform and/or of any person’s use of the Site and/or Platform, and shall not be responsible for any illegal use of the Site and/or Platform by You. It is Your responsibility to ensure that You comply with any and all laws applicable to You before using the Site and/or Platform. If You have any concerns, You should consult with your own legal counsel in the applicable jurisdiction about the legality of Your use of the Site and/or Platform. The Company reserves the right at any time to request from You evidence of age and if satisfactory proof of age is not provided within 72 hours of requesting such proof, then You shall be prohibited from using the Site and Platform. In the event that it is found that You are under the age of 18, or the legal age as determined by the laws of the country where You live (whichever is higher), then the Company shall cease Your use of the Site and Platform." /></li>
+      </ol>
+    </div>
+  );
+
+  renderRepresentations = () => (
```

Comment:
This should be a component, not a method. Make a stateless component please.

Reply:
done